src/components/buttons/primary-action-button/primary-action-button.js

```diff
@@ -0,0 +1,96 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import filterDataAttributes from '../../../utils/filter-data-attributes';
+import filterAriaAttributes from '../../../utils/filter-aria-attributes';
```

Comment:
This import occurs twice.

Reply:
Which one? one is for Aria and the other one for Data

Reply:
Ah my bad. All good then!